assets/examples/infrastructure/newrelic-infra-template.yml.example

```diff
@@ -685,6 +685,18 @@ license_key: your_license_key
 #startup_connection_retries: 6
 #
 
+#
+# Option   : logging_retry_limit
+# Env var  : NRIA_LOGGING_RETRY_LIMIT
+# Value    : Number of retries the logging forwarder should attempt. 
+#            Integer values are for the number of intended retries. 
+#            Other possible values include `False` to set the number of retries to infinite 
+#            and `no_retries` to turn off the retry functionality entirely.
+# Default  : 5s
```

Comment:
picky comment, the default value has an `s` ☝️ 😅

Reply:
Ugh, thanks for catching that. Fixed and pushed another commit.